<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuarquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionSalvarComo"/>
    <addaction name="separator"/>
    <addaction name="actionConfigurarPagina"/>
    <addaction name="actionVisualizarImpressao"/>
    <addaction name="actionImprimir"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionDesfazer"/>
    <addaction name="actionRefazer"/>
    <addaction name="separator"/>
    <addaction name="actionRecortar"/>
    <addaction name="actionCopiar"/>
    <addaction name="actionColar"/>
    <addaction name="actionExcluir"/>
   </widget>
   <widget class="QMenu" name="menuFormatar">
    <property name="title">
     <string>Formatar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExibir">
    <property name="title">
     <string>Exibir</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuarquivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuFormatar"/>
   <addaction name="menuExibir"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionColar">
   <property name="text">
    <string>Colar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRecortar">
   <property name="text">
    <string>Recortar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSalvarComo">
   <property name="text">
    <string>Salvar Como</string>
   </property>
  </action>
  <action name="actionConfigurarPagina">
   <property name="text">
    <string>Configurar Página</string>
   </property>
  </action>
  <action name="actionImprimir">
   <property name="text">
    <string>Imprimir</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionExcluir">
   <property name="text">
    <string>Excluir</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRefazer">
   <property name="text">
    <string>Refazer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionDesfazer">
   <property name="text">
    <string>Desfazer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionVisualizarImpressao">
   <property name="text">
    <string>Visualizar Impressão</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
